ATTACK/Assets/Scripts/Colors.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class Colors
+{
+
+    public static Color LightPurple = new Color(0.36f, 0.36f, 0.67f);
```

Comment:
All of these fields should probably be readonly, e.g. public static readonly Color LightPurple = ...

Reply:
True, or just const?